src/core/infrastructure/entities/Message.ts

```diff
@@ -0,0 +1,19 @@
+import { Column, Entity, Unique, JoinColumn, ManyToOne, PrimaryGeneratedColumn } from "typeorm";
+import { UserRecord } from "./User";
+
+@Entity({ name: "tribe_messages" })
+@Unique(["id"])
```

Comment:
You don't need to specify this because of `@PrimaryGeneratedColumn("uuid", { name: "id" })`